bin/crew

```diff
@@ -32,7 +32,7 @@ Usage:
   crew reinstall [options] [-k|--keep] [-s|--build-from-source] [-S|--recursive-build] <name> ...
   crew remove [options] <name> ...
   crew search [options] [<name> ...]
-  crew update [options]
+  crew update [options] [<compatible>]
```

Comment:
Why do we need this?

Reply:
Without this you can't test `generate_compatible` without first putting your crew changes up on a git repo!

Essentially it saves a great deal of time in the debug cycle.

Reply:
I guess I don't understand why it's important for development.